transit_odp/common/password_validation.py

```diff
@@ -0,0 +1,84 @@
+import re
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext
+
+
+class CustomPasswordValidator:
+    """
+    Custom password validator for Django authentication system.
+
+    This validator checks if a password meets the following criteria:
+    - Contains at least one uppercase letter.
+    - Contains at least one lowercase letter.
+    - Contains at least one digit.
+    - Contains at least one special character from the set: ~!@#$%^&*()_+{}":;'<>\?/|
+
+    """
+
+    def validate(self, password, user=None):
+        """
+        Validate the password against the defined criteria.
+
+        Args:
+        - password (str): The password to validate.
+        - user (object, optional): The user object (default: None).
+
+        Raises:
+        - ValidationError: If the password does not meet any of the validation criteria.
+        """
+
+        # Minimum password length
+        min_length = 8
```

Comment:
Add this in settings

Reply:
minimum length validation has been removed from this custom validator 
as it is already added in base.py